template.yaml

```diff
@@ -299,14 +299,14 @@ Resources:
       Environment:
         Variables:
           ALLOWED_ORIGIN: '*'
-          TABLE_NAME: !Ref PublicationsTableName
+          TABLE_NAME: !Ref PublicationsTable
```

Comment:
Maybe we should make the table name and index names internal to simplify stack setup?

EDIT: I guess it helps with default values.

Reply:
yes, using defaults without overriding them is poor man's constants